u2f-chrome-extension/u2f-api.js

```diff
@@ -88,6 +92,19 @@ u2f.Response;
  */
 u2f.Error;
 
+/**
+ * Data object for a single sign request.
+ * @typedef {enum {BLUETOOTH_RADIO, BLUETOOTH_LOW_ENERGY, USB, NFC}}
+ */
+u2f.Transport;
+
+
+/**
+ * Data object for a single sign request.
+// * @typedef {sequence<u2f.Transport>}
```

Comment:
As far as I know, there is no sequence construct in javascript. I was unsure about sequence vs. Array, as the previous API calls it Array and the new one sequence. I checked with Alexei and Nick and in the end concluded this should just be array. Then I forgot to delete the sequence line. What should it be?